src/openrct2/ride/Vehicle.cpp

```diff
@@ -2443,6 +2443,16 @@ void Vehicle::UpdateWaitingToDepart()
         }
     }
 
+    // Set random speed for rider controlled vehicles
+    if (GetRideEntry()->flags & RIDE_ENTRY_FLAG_RIDER_CONTROLS_SPEED && num_peeps != 0)
+    {
+        // There are 64 possible settings for the rider speed preferences - pick one at random
+        rider_speed_preference = (scenario_rand() & 0x3F);
```

Comment:
I would instead tie it to the sprite_index of the rider. No need to make it random and then riders will be consistently be slow/fast